src/org/usfirst/frc2175/config/IntakeConfig.java

```diff
@@ -0,0 +1,49 @@
+package org.usfirst.frc2175.config;
+
+import java.util.Properties;
+
+public class IntakeConfig extends BaseConfig {
+    private static final String PROPERTY_FILE_NAME = "intake.properties";
+    private double sideBeltSpeedForward;
+    private double sideBeltSpeedReverse;
+    private double mainBeltSpeedForward;
+    private double mainBeltSpeedReverse;
+
+    @Override
+    public String getPropertyFileName() {
+        return PROPERTY_FILE_NAME;
+    }
+
+    @Override
+    protected void configure(Properties properties) {
+        int sideBeltSpeedForward = getIntPropertyValue(
```

Comment:
You made all of these variables ints instead of doubles. You're saving the values as doubles eventually anyway, so these should be switched to `double` instead of `int`.

Also, making it `int` here means that you can only choose 1, 0, or -1 for the intake speed. ;)